src/lib.rs

```diff
@@ -20,28 +20,28 @@
 //! }
 //! ```
 
-
 extern crate failure;
-#[cfg(windows)]
-extern crate winapi;
-#[cfg(windows)]
-extern crate widestring;
 #[cfg(unix)]
 extern crate libc;
+extern crate nix;
```

Comment:
missing `#[cfg(unix)]` here, cause build on windows failed.

Reply:
Whoops. I always forget that `cfg` doesn't carry through a block of statements. Sorry, will fix tonight.